database_schemas/cassandra/benchflow.cql

```diff
@@ -46,6 +46,7 @@ CREATE TABLE environment_data (
   container_id text,
   read_time text,
   cpu_percpu_usage list<bigint>,
+  cpu_percpu_percent_usage list<double>,
```

Comment:
Why are these percentage of the `double` type, while the `cpu_percent_usage` is a `float`?